### DISCUSSION TITLE
Ran WGCNA! Interpretation and next steps?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1195] [Q&A] [answered]
Alright, so I was able to complete running WGCNA on my ambient-temperature crabs over days 0, 2, and 17. These libraries are aligned to cbai_transcriptomev2.0, which is not filtered by taxa, and thus it examines both C. bairdi and Hematodinium genes together

To speed the process up, I rejected genes with counts below 30 in all samples (recommendation is below 10), so these aren't perfect, but they'll be decent! 

The heat map is particularly interesting - seems to not show any significant relationships with day (if we set the p-value bar at 0.05), but some interesting relationships with individual crab.

I could really use some help both interpreting these results and deciding where to go next. 

- Recreate this with Day 0/2 Elevated crabs

- Recreate this with Day 2 Ambient/Day 2 Elevated crabs

- Continue by performing GO-MWU on each WGCNA module

Any thoughts on what the best way to go from here would be?

[Sample dendrogram](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/ClusterDendrogram.png)

[Sample dendrogram with condition](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/ClusterDendrogram_W_Colors.png)

[Cluster dendrogram with original and merged eigengenes](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/ClusterDendrogramOrigAndMergedEigengenes.png)

[Clustering of module eigengenes](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/ClusteredEigengenes.png)

[Gene clustering on TOM-based dissimilarity](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/GeneDendrogram.png)

[Gene dendrogram with module colors](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/GeneDendrogramWColors). NOTE: Didn't work at first, as of time of posting this link is broken - will re-run soon, thus hopefully fixing the link

[Mean connectivity](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/MeanConnectivity.png)

[Heat map of module-treatment relationships](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/ModuleTreatmentHeatMap.png)

[Scale independence](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/ScaleIndependence.png)

[Specific gene lists for each eigengene](https://github.com/afcoyle/hemat_bairdi_transcriptome/tree/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30)

**Comment by sr320** [comment, marked as answer]
sorry if I am missing but is there a image showing expression patterns over 3 days?

**Reply by aspencoyle** [reply]
Shoot, no there isn't - I based my analysis off a script that didn't look at expression patterns over time, but one of the tutorials I finished earlier did do that. I'll start on that next - hopefully should be relatively quick!

**Reply by aspencoyle** [reply]
Hmm, update - apparently I misremembered, there weren't any tutorials that dealt with time series data or graphing single-gene differences in expression patterns (which is what I assumed you were asking for)

If you're just looking for an image that shows expression patterns in each eigengene cluster over time, [this might be what you're looking for - the heat map](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver30/ModuleTreatmentHeatMap.png).

It only shows differences in expression over time, without differentiating between day 0/2/17. However, I believe I could separate the days out - like in the graph at the bottom of [this page](https://wikis.utexas.edu/display/bioiteam/Clustering+using+WGCNA) if that's what you mean!

Furthermore, I ran WGCNA again, this time changing the bar from "must be >30 counts in 1+ of our 9 samples" to "must be >10 counts in 2+ of our 9 samples", as it seemed closer to the recommendation (must be >10 counts in 2+ of 10+ samples). 

That produced a heat map that [isn't too different](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/WGCNA_output/AmbCrabs_cbai_transcriptome_v2.0_trial/OnlyCtsOver10/ModuleTreatmentHeatMap.png), but thought I'd mention it!

**Comment by sr320** [comment]
Note you can put images in posts 😄 

I was thinking something more like 
![this](http://gannet.fish.washington.edu/seashell/snaps/Screen_Shot_on_2021-05-03_at_09-31-43.png__Droplr_2021-05-03_09-32-41.png)
